assets/js/base/components/payment-methods/payment-method-options.js

```diff
@@ -42,50 +44,41 @@ const PaymentMethodOptions = () => {
 	const { removeNotice } = useStoreNotices();
 	const { isEditor } = useEditorContext();
 
+	const options = Object.keys( paymentMethods ).map( ( name ) => {
+		const { edit, content, label, supports } = paymentMethods[ name ];
+		const component = isEditor ? edit : content;
+		return {
+			value: name,
+			label:
+				typeof label === 'string'
+					? label
+					: cloneElement( label, {
+							components: paymentMethodInterface.components,
+					  } ),
+			name: `wc-saved-payment-method-token-${ name }`,
+			content: (
+				<PaymentMethodTab allowsSaving={ supports.savePaymentInfo }>
+					{ cloneElement( component, {
+						activePaymentMethod,
+						...paymentMethodInterface,
+					} ) }
+				</PaymentMethodTab>
+			),
+		};
+	} );
+
+	const updateToken = ( value ) => {
+		onChange( value );
+		setActivePaymentMethod( value );
+		removeNotice( 'wc-payment-error', noticeContexts.PAYMENTS );
+	};
+
 	return expressPaymentMethodActive ? null : (
-		<Tabs
-			className="wc-block-components-checkout-payment-methods"
-			onSelect={ ( tabName ) => {
-				setActivePaymentMethod( tabName );
-				removeNotice( 'wc-payment-error', noticeContexts.PAYMENTS );
-			} }
-			tabs={ Object.keys( paymentMethods ).map( ( name ) => {
-				const {
-					ariaLabel,
-					edit,
-					content,
-					label,
-					supports,
-				} = paymentMethods[ name ];
-				const component = isEditor ? edit : content;
-				return {
-					name,
-					title:
-						typeof label === 'string'
-							? label
-							: cloneElement( label, {
-									components:
-										paymentMethodInterface.components,
-							  } ),
-					ariaLabel,
-					content: (
-						<PaymentMethodTab
-							allowsSaving={ supports.savePaymentInfo }
-						>
-							{ cloneElement( component, {
-								activePaymentMethod,
-								...paymentMethodInterface,
-							} ) }
-						</PaymentMethodTab>
-					),
-				};
-			} ) }
-			initialTabName={ activePaymentMethod }
-			ariaLabel={ __(
-				'Payment Methods',
-				'woo-gutenberg-products-block'
-			) }
-			id="wc-block-payment-methods"
+		<RadioControlAccordion
+			id={ 'wc-payment-method-options' }
+			selected={ selectedMethod }
```

Comment:
You'll likely want to use `activePaymentMethod` here (ultimately powered by context - `PaymentMethodDataContext`). 

That gives you the defaulting behaviour (ensure a payment method is selected at all times). 

I hacked this change in my local dev and it fixes the missing default on load, and also handles if payment methods disappear at runtime (e.g. COD dependent on shipping). 

![Screen Capture on 2020-11-25 at 15-22-23](https://user-images.githubusercontent.com/4167300/100174905-3d679180-2f32-11eb-8572-81687b564665.gif)

Reply:
I've pushed a fix for this: 5381cfc1828afb61a4b5fed5d61f8788863f98f1